src/FritzBox/Converter.php

```diff
@@ -176,6 +189,16 @@ private function getPhoneNumbers()
                 }
             }
         }
+        if (count($phoneNumbers) > 1) {
+            usort($phoneNumbers, function($a, $b) {
+                $idx1 = array_search($a['type'], $this->phoneSort, true);
+                $idx2 = array_search($b['type'], $this->phoneSort, true);
+                if ($idx1 == $idx2)
+                    return ($a['number'] > $b['number']) ? 1 : -1;
+                else
+                    return ($idx1 < $idx2) ? -1 : +1;
```

Comment:
Nur so aus Konsistenzgeünden: warum nicht beide vergleiche einfach auch genau gleich machen? Mit gleichen operator, gleichen vorzeichen?

Reply:
Sorry wenn ich da in die falsche richtung geführt habe